tensorflow/core/kernels/cwise_ops_sycl_common.h

```diff
@@ -154,6 +154,28 @@ struct BinaryFunctor<SYCLDevice, Functor, NDIMS, has_errors> {
   DEFINE_BINARY5(F, T5, T6, T7, T8, T9)
 
 }  // end namespace functor
+
+
+#if defined(__ANDROID_TYPES_SLIM__)
```

Comment:
Why is this needed for coefficient wise operations and not for all the other sycl kernels ?